### PR TITLE
add bytedance/tiktok

### PR DESCRIPTION
**Pull request by dingelish**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -38,6 +38,27 @@
       }
     }
   },
+  "ByteDance/Tiktok": {
+    "url": "https://job.toutiao.com/s/J7LBfB5",
+    "jobs": {
+      "softwareEngineer": {
+        "intern (referral code: VC7FSRM)": {
```
Comment (CatChen) [inline data.json:45]: `intern` and `newGrad` are special keys. If the keys don't match the data won't show up at all.

File: data.json [hunk 2]
```diff
@@ -38,6 +38,27 @@
...
+          "available": true,
+          "url": "https://careers.tiktok.com/position?project=6854152751179303182&type=3&current=1&limit=100"
+        },
+        "newGrad (referral code: VC7FSRM)": {
+          "available": true,
+          "url": "https://careers.tiktok.com/position?project=6854152751179303182&type=3&current=1&limit=100"
```
Comment (CatChen) [inline data.json:51]: These are intern positions instead of new grad positions. Do you have a different link for new grad positions?

**Review by CatChen — CHANGES_REQUESTED** [review]

**Comment by CatChen** [comment]
Thanks for your contribution! We plan to support internal referrals in the future. We haven't decided how to facilitate internal referrals while keeping it fair and engaging yet. So sorry about not being able to support showing referral code at this point.

**Comment by dingelish** [comment]
resolved in 3a2e130 thanks!

**Review by CatChen — APPROVED** [review]
Thank you!

**Pull request merged**